app/screens/find_channels/filtered_list/filtered_list.tsx

```diff
@@ -244,12 +244,19 @@ const FilteredList = ({
             items.push(...usersMatchStart);
         }
 
+        // Archived channels local
+        if (items.length < MAX_RESULTS) {
+            const archivedAlpha = archivedChannels.
+                sort(sortChannelsByDisplayName.bind(null, locale));
+            items.push(...archivedAlpha.slice(0, MAX_RESULTS + 1));
+        }
+
         // Remote Channels that start with
         if (items.length < MAX_RESULTS) {
             items.push(...remoteChannels.startWith);
         }
 
-        // Users & Channels that matches
+        // // Users & Channels that matches
```

Comment:
Unneeded change?

Reply:
@larkox currently without internet at home, would you mind removing the extra // :)

Thanks